templates/data-sources/fabric_precision_time.md.tmpl

```diff
@@ -0,0 +1,22 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "equinix_fabric_precision_time Data Source - terraform-provider-equinix"
+subcategory: "Fabric"
+description: |-
+  Fabric V4 API compatible data resource that allow user to fetch Equinix Fabric Precision Time Service by Uuid
+---
+
+# equinix_fabric_precision_time (Data Source)
+
+Fabric V4 API compatible data resource that allow user to fetch Equinix Fabric Precision Time Service by Uuid
+
+Additional documentation:
+* Getting Started: https://docs.equinix.com/en-us/Content/Edge-Services/EPT/EPT.htm
+* API: https://developer.equinix.com/dev-docs/fabric/api-reference/fabric-v4-apis#precision-time
```

Comment:
You could instead put this in the `Description` or `MarkdownDescription` attribute of the data source schema and reference `{{ .Description | trimspace }}` in the docs template.  This would keep the description in the docs while also making available to other tools.